lib/rubocop/cop/lint/useless_else_without_rescue.rb

```diff
@@ -4,6 +4,7 @@ module RuboCop
   module Cop
     module Lint
       # This cop checks for useless `else` in `begin..end` without `rescue`.
+      # And the syntax does not support Ruby 2.6 or higher.
```

Comment:
Maybe we should say here "Note: This syntax is no longer valid on Ruby 2.6 or higher and this cop is going to removed at some point the future."?

Reply:
Yes, quite right! I updated it.